### PR TITLE
doc: Update docker page

### PR DESCRIPTION
**Pull request by shimonp21**
#### Summary

Nice for the future: dynamically put the latest `cli` version in the snippet instead of using `latest`, which is  a bit buggy with docker...

### CODE REVIEW

File: website/pages/docs/advanced-topics/docker.md [hunk 1]
```diff
@@ -10,10 +10,34 @@ It is possible to use CloudQuery in an isolated container. You can pull the rele
...
+### Create a Config File on the Host Machine
+
+CloudQuery uses YAML files as the primary means of configuration. A simple example `config.yml` file would look like:
+
+```yaml
+kind: source
+spec:
+  name: aws
+  path: cloudquery/aws
+  version: "v8.0.0"
```
Comment (bbernays) [inline website/pages/docs/advanced-topics/docker.md:22]: Should this use the variable replacement?

File: website/pages/docs/advanced-topics/docker.md [hunk 2]
```diff
@@ -46,3 +72,11 @@ Depending on your operating system, the built components maybe different between
 
 </Callout>
 
+```
```
Comment (erezrokah) [inline website/pages/docs/advanced-topics/docker.md:75]:
```suggestion
```bash
```

File: website/pages/docs/advanced-topics/docker.md [hunk 3]
```diff
@@ -10,10 +10,34 @@ It is possible to use CloudQuery in an isolated container. You can pull the rele
...
+    connection_string: "postgres://postgres:pass@host.docker.internal:5432/postgres?sslmode=disable"
+```
 
-```docker copy
-docker run \
+### Run the Container
+
+For the CloudQuery docker container to use this configuration file you will need to mount the volume to the container like so:
+
+```docker
```
Comment (erezrokah) [inline website/pages/docs/advanced-topics/docker.md:39]:
```suggestion
```bash 
```

File: website/pages/docs/advanced-topics/docker.md [hunk 4]
```diff
@@ -30,7 +56,7 @@ If you are running Docker on an ARM Apple device and you see a segmentation faul
 
 ## Caching
 
-Due to the way `cloudquery` is [architected](/docs/developers/architecture) it downloads all the components to interact with source and destination plugins. This means that with a docker container it runs the download step each state as the local cache is lost between executions. To avoid this we recommend mounting a volume to cache the data and configuring `cloudquery` to use this via the `--data-dir` optional flag. An example of this would be:
+Due to the way `cloudquery` is [designed](/docs/developers/architecture) it downloads all the components to interact with source and destination plugins. This means that with a docker container it runs the download step each time, as the local cache is lost between executions. To avoid this we recommend mounting a volume to cache the data and configuring `cloudquery` to use this via the `--cq-dir` optional flag. An example of this would be:
 
 ```docker copy
```
Comment (erezrokah) [inline website/pages/docs/advanced-topics/docker.md:61]:
```suggestion
```bash copy
```

**Review by erezrokah — APPROVED** [review]
Looks good but we might want to update the code blocks language

File: website/pages/docs/advanced-topics/docker.md (hunk 1 above)
Comment (erezrokah) [inline website/pages/docs/advanced-topics/docker.md:22]:
```suggestion
  version: "VERSION_SOURCE_AWS"
```

**Review by bbernays — APPROVED** [review]

**Pull request merged**